src/Symfony/Component/Validator/CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ CHANGELOG
  * deprecated `Symfony\Component\Validator\Mapping\Cache\CacheInterface` in favor of PSR-6.
  * deprecated `ValidatorBuilder::setMetadataCache`, use `ValidatorBuilder::setMappingCache` instead.
  * Marked the `ValidatorDataCollector` class as `@final`.
+ * added the `Hostname` constraint and validator
```

Comment:
this needs to be moved to a `5.1` section

Reply:
this PR is now against symfony:4.4. Should I rebase it against symfony:master?

Reply:
please do so

Reply:
ok, done